lib/view_model/settings/other_settings_view_model.dart

```diff
@@ -81,13 +94,25 @@ abstract class OtherSettingsViewModelBase with Store {
 
   String getBuyProviderType(dynamic buyProviderType) {
     final _buyProviderType = buyProviderType as BuyProviderType;
+    return _buyProviderType == BuyProviderType.askEachTime
+        ? S.current.ask_each_time
+        : _buyProviderType.name;
```

Comment:
**Minor:**
can be directly `.name` since `askEachTime` also has `name`

Reply:
Can't use localization directly in the name extension for ask_each_time due to late initialization issues.